xml/docu_styleguide.structure.xml

```diff
@@ -2535,6 +2535,74 @@ xml:id="tab-install-source"</screen>
 <screen>&lt;phrase role="productname"&gt;LibreOffice&lt;/phrase&gt; is an office suite.</screen>
  </sect2>
 
+ <sect2 xml:id="sec-profiling">
+  <title>Profiling</title>
+  <para>
+   Profiling is convenient for the creation of consistent documentation 
+   across different products or product lines. This is especially beneficial 
+   when similar products share a considerable amount of features, with only 
+   minor differences. Instead of maintaining separate documentation for
+   each product, you can share most of the XML source code and only vary text 
+   snippets where necessary. In DocBook XML files, you can mark some elements
+   as conditional by using profiling attributes. When processing the files to 
+   generate output, specify which conditions apply to the output. The style sheets
```

Comment:
I think the order is problematic: you _first_ specify the conditions, and _then_ you process the files. Not the other way around. The order should be reversed:

```suggestion
   as conditional by using profiling attributes. Specify which conditions
   apply to the output when processing the files to generate output. The style sheets
```